packages/core/src/__tests__/getStateFromPath.test.tsx

```diff
@@ -144,7 +144,6 @@ it('converts path string to initial state with config with nested screens', () =
   const path = '/foe/bar/sweet/apple/baz/jane?count=10&answer=42&valid=true';
   const config = {
     Foo: {
-      path: 'foo',
```

Comment:
Do we need to change existing tests? This should be an additional feature, right?

Reply:
Yeah, I just did it for clarity that it is a navigator and there should be no need to navigate to it explicitly, but it doesn't break anything.